server/sonar-server/src/main/java/org/sonar/server/projectanalysis/ws/CreateEventAction.java

```diff
@@ -113,18 +113,13 @@ public void handle(Request httpRequest, Response httpResponse) throws Exception
   private CreateEventResponse doHandle(CreateEventRequest request) {
     try (DbSession dbSession = dbClient.openSession(false)) {
       SnapshotDto analysis = getAnalysis(dbSession, request);
-      checkPermissions(analysis);
       checkExistingDbEvents(dbSession, request, analysis);
       EventDto dbEvent = insertDbEvent(dbSession, request, analysis);
       return toCreateEventResponse(dbEvent);
     }
   }
 
-  private void checkPermissions(SnapshotDto analysis) {
-    userSession.checkComponentUuidPermission(UserRole.ADMIN, analysis.getComponentUuid());
-  }
-
-  private EventDto insertDbEvent(DbSession dbSession, CreateEventRequest request, SnapshotDto analysis) {
+    private EventDto insertDbEvent(DbSession dbSession, CreateEventRequest request, SnapshotDto analysis) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Make this line start at column 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)